installer/manifests/keptn/templates/rbac.yaml

```diff
@@ -8,14 +8,17 @@ metadata:
 ---
 apiVersion: v1
 kind: ServiceAccount
+metadata:
+  name: keptn-resource-service
+  labels: {{- include "keptn.common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/name: resource-service
+---
+apiVersion: v1
+kind: ServiceAccount
 metadata:
   name: keptn-configuration-service
```

Comment:
We need to keep this in the next version to support Zero Downtime upgrades - otherwise the pod running the v0.17 resource service will not be able to access any git upstream credentials during the upgrade from 0.17 to 0.18. After the release of 0.18, this service account and its associated role bindings should be removed